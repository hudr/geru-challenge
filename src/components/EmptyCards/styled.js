import styled from 'styled-components'

export const StyledEmptyContainer = styled.div`
  width: 100%;
  height: 100%;
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
`

export const NotesImg = styled.img`
  width: 40vh;
  height: 40vh;
`

export const Subtitle = styled.h3`
  color: grey;
`
