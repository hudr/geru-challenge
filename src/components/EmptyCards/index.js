import React, { Component, Fragment } from 'react'

import { StyledEmptyContainer, NotesImg, Subtitle } from './styled'

import emptyImage from '../../assets/images/nodata.svg'

class EmptyCards extends Component {
  render() {
    return (
      <Fragment>
        <StyledEmptyContainer>
          <NotesImg src={emptyImage} alt='emptyimage' />
          <Subtitle>Your todo list is empty :)</Subtitle>
        </StyledEmptyContainer>
      </Fragment>
    )
  }
}

export default EmptyCards
