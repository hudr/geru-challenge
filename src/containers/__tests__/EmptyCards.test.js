import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import EmptyCards from '../../components/EmptyCards'

describe('<EmptyCards />', () => {
  it('renders the component', () => {
    const wrapper = shallow(<EmptyCards />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
