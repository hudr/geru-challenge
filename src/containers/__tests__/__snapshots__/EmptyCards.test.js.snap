// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EmptyCards /> renders the component 1`] = `
<Fragment>
  <styled.div>
    <styled.img
      alt="la"
      src="nodata.svg"
    />
    <styled.h3>
      Your todo list is empty :)
    </styled.h3>
  </styled.div>
</Fragment>
`;
